### ISSUE TITLE
Zero-byte reproducers since yesterday

### ISSUE THREAD

**Issue by rurban** [google/oss-fuzz#5386]
I get zero-byte reproducers in my project libredwg since yesterday, a regression.
But not in all cases. I got proper testcases also since.
Since yesterday: 2 bad, one good case.

The bug description and stack trace looks correct, seems to be a real bug.
It's labelled as unminimized.

e.g. https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=32048&q=libredwg
https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=32022&q=libredwg

**Comment by inferno-chromium** [comment]
So these are unminimized and unreproducible as running an individual testcase is not reproducing the leak. If you just run the fuzzer, you should run into it quickly and reliably (reliably because Fuzz-Blocker which means most iterations of fuzzer running into it). Some regression in your code, might have to debug what introduced it.

**Comment by rurban** [comment]
Wrong. Empty reproducers are certainly an oss-fuzz bug, probably with a recent change in the minimizer.
The stack trace looks correct, and empty files don't make it that far.

**Comment by inferno-chromium** [comment]
no recent clusterfuzz/oss-fuzz changes went in.

**Comment by rurban** [comment]
https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=32094&q=libredwg

came in, and also has zero bytes. Also see GH #5388

**Comment by oliverchang** [comment]
These Fuzz-Blocker issues with zero bytes usually indicate that we run into them very frequently during fuzzing. While they may not consistently reproducible with the given empty input, it usually points to some non-determinism or state that gets built up during a fuzzing run as testcases are run repeatedly.

Could you try just running the fuzzer with libFuzzer (without any inputs) and see if it crashes in the same way after a few minutes?

**Comment by rurban** [comment]
yeah, that's what I'm doing. but unfortunately I didn't get into these bugs yet, only into other.
thanks for confirmation.

**Comment by a1xndr** [comment]
I seem to be running into something similar with a recent qemu crash:
For example:
gs://qemu-logs.clusterfuzz-external.appspot.com/libFuzzer_qemu_qemu-fuzz-i386-target-generic-fuzz-am53c974/libfuzzer_asan_qemu/2021-03-24/13:51:24:576191.log
On my local-machine triggering the same stack-overflow requires a couple hundred-byte inputs. And not a single crash- file found locally is 0 bytes long. However, the corresponding crash saved by clusterfuzz is 0 bytes long:
gs://qemu-logs.clusterfuzz-external.appspot.com/libFuzzer_qemu_qemu-fuzz-i386-target-generic-fuzz-am53c974/libfuzzer_asan_qemu/2021-03-24/13:51:24:576191.testcase 
https://oss-fuzz.com/testcase-detail/4595666119688192 - of course the crash is marked not reproducible and doesn't end up on monorail

For this fuzzer, it is very unlikely that there is state leaking anywhere. Moreover, for the same target, OSS-Fuzz properly reports an abort issue, and provides a real reproducer. Apologies if this is unrelated to this report...
